File: examples/python/openai_client.py
from openai import OpenAI

client = OpenAI(
    api_key="sk-change-me",
    base_url="http://127.0.0.1:8080/v1",
)

response = client.chat.completions.create(
    model="balanced",
    messages=[
        {"role": "system", "content": "You are a concise assistant."},
        {"role": "user", "content": "List two safe rollout checks."},
    ],
)

print(response.choices[0].message.content)

File: examples/node/openai-client.mjs
import OpenAI from "openai";

const client = new OpenAI({
  apiKey: "sk-change-me",
  baseURL: "http://127.0.0.1:8080/v1",
});

const completion = await client.chat.completions.create({
  model: "creative",
  messages: [
    { role: "system", content: "You are a concise assistant." },
    { role: "user", content: "Give me a short release announcement." },
  ],
});

console.log(completion.choices[0].message.content);
